rs/boundary_node/rate_limits/canister/metrics.rs

```diff
@@ -36,3 +121,104 @@ pub fn serve_metrics(
         }
     }
 }
+
+pub fn recompute_metrics() {
+    STABLE_MEMORY_SIZE.with(|cell| {
+        let memory = (ic_cdk::api::stable::stable_size() * WASM_PAGE_SIZE_IN_BYTES) as f64;
+        cell.borrow_mut().set(memory);
+    });
+
+    API_BOUNDARY_NODES_COUNT.with(|cell| {
+        cell.borrow_mut()
+            .set(API_BOUNDARY_NODE_PRINCIPALS.with(|cell| cell.borrow().len() as i64));
+    });
+
+    ACTIVE_VERSION.with(|cell| {
+        cell.borrow_mut().set(
+            CONFIGS.with(|cell| cell.borrow().last_key_value().map_or(0, |(key, _)| key)) as i64,
+        );
+    });
+
+    CONFIGS_COUNT.with(|cell| {
+        cell.borrow_mut()
+            .set(CONFIGS.with(|cell| cell.borrow().len() as i64));
+    });
+
+    INCIDENTS_COUNT.with(|cell| {
+        cell.borrow_mut()
+            .set(INCIDENTS.with(|cell| cell.borrow().len() as i64));
+    });
```

Comment:
Why don't you use state::incidents_count

Reply:
corrected, see above